src/configure/model/models.ts

```diff
@@ -35,6 +35,7 @@ export class WizardInputs {
     azureSession: AzureSession;
     subscriptionId: string;
     githubPATToken?: string;
+    potentialTemplates?: PipelineTemplate[];
```

Comment:
Why does it need to be a list of templates? It should just a `templateLabel`. And we should maintain a seprate templateMapper which maps a label to _n_ templates

Reply:
The list of potential templates stores the templates (both local and remote as well as both linux/windows, depending upon the user selection of the template label). Later, when the user selects the resource, we decide the final template.